Cargo.toml

```diff
@@ -13,3 +13,10 @@ polars-core = { version = "0.41.0", default-features = false }
 polars-ffi = { version = "0.41.0", default-features = false }
 polars-plan = { version = "0.41.0", default-feautres = false }
 polars-lazy = { version = "0.41.0", default-features = false }
+
+[patch.crates-io]
```

Comment:
Can we point this to the main branch until `0.42` is released.

Reply:
Cargo only recognizes patches defined in root project and not in dependencies. `pyo3-polars` is meant to be a dependency of projects that use it. So unless they declare patches in their `Cargo.toml`, patches are not in effect.

Patches declared here is only for testing. We can only wait for `0.42`.

Reply:
Alright. 0.42 is planned in ca. 2 weeks.